<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Echellonnement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>37</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox{
	background-color: rgba(191, 191, 191, 0.5);
	color: #000;
	border: 2px solid darkgray;
}
</string>
           </property>
           <property name="title">
            <string>Ajout dans la liste d'attente</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(25, 25, 25);</string>
              </property>
              <property name="text">
               <string>Identifiant :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ajout_inp_id">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(18, 18, 18);</string>
              </property>
              <property name="text">
               <string>Date de demande :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="ajout_inp_date">
              <property name="styleSheet">
               <string notr="true">QDateEdit{
	background-color: #fff;
	color: #000;
}</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(20, 20, 20);</string>
              </property>
              <property name="text">
               <string>Piece justificative :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="ajout_inp_piece">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="ajouter_btn">
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	background-color: rgb(0, 113, 0);
	height: 25px;
	width: 80px;
	font: 81 italic 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
	color: #fff;
}

QPushButton:pressed{
	background-color: rgb(0, 85, 0);
}</string>
                </property>
                <property name="text">
                 <string>Ajouter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer_3">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(0, 113, 0);
	height: 25px;
	width: 80px;
	font: 81 italic 12pt &quot;Fira Sans Compressed ExtraBold&quot;;
}

QPushButton:pressed{
	background-color: rgb(0, 85, 0);
}</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>7</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox{
	background-color: rgba(191, 191, 191, 0.5);
	color: #000;
	border: 2px solid darkgray;
}</string>
           </property>
           <property name="title">
            <string>Autorisation</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color: rgb(20, 20, 20);</string>
              </property>
              <property name="text">
               <string>Identifiant :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="autorisation_inp_id">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="autorisation_btn">
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	background-color: rgb(0, 113, 0);
	height: 25px;
	width: 80px;
	font: 81 italic 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
	color: #fff;
}

QPushButton:pressed{
	background-color: rgb(0, 85, 0);
}</string>
                </property>
                <property name="text">
                 <string>Confirmer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>7</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="vision_liste_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	height: 25px;
	font: 81 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
}</string>
           </property>
           <property name="text">
            <string>Visionner la liste des demandes</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QTableView" name="vue">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView{
	border: 2px solid rgb(85, 85, 127);
	background-color: #fff;
	color: black;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>258</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	height: 25px;
	font: 81 Italic 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
	color: black;
	background-color: red;
}

QpushButton:pressed{
	background-color: rgb(182, 7, 7);
}</string>
         </property>
         <property name="text">
          <string>Retour</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>37</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
